### DISCUSSION TITLE
mspider/user-guide/mesh-config/istio-resources/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1312] [Announcements]
# mspider/user-guide/mesh-config/istio-resources/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/mesh-config/istio-resources/

**Comment by liyuerich** [comment]
TLS密钥管理能否增加一下文档？

**Comment by hanxiaop** [comment]
telementry 拼写错误，而且需要大写，Telemetry
